packages/commonwealth/client/scripts/views/pages/proposals.tsx

```diff
@@ -50,6 +50,12 @@ const ProposalsPage = () => {
   const [isSubstrateLoading, setSubstrateLoading] = useState(false);
   useInitChainIfNeeded(app); // if chain is selected, but data not loaded, initialize it
 
+  const onSubstrate = app.chain && app.chain.base === ChainBase.Substrate;
```

Comment:
we can shorten it using question mark operator `app.chain?.base === ChainBase.Substrate`

Reply:
done for all of these